src/ToDo.Wasm/ToDo - Backup.Wasm.csproj

```diff
@@ -0,0 +1,94 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Is this file still needed?

Reply:
Hummm I didn't saw this one, and actually I don't even know where this file is coming from :/ Going to remove it.